### PR TITLE
Added tests readme

### PR DESCRIPTION
**Pull request by OGKevin**
Adds readme for tests.

Running tests from command line seems to be a pain in the but tho 😢.

### CODE REVIEW

File: src/test/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Besides these scenarios, some code of ApiContext, ApiClient and the JSON module 
+are also tested :thumbs_up:.
+
+## Configuration
+
+To run the tests you must first setup the test configuration JSON. The example
+of a configuration file is located at [`src/test/config.example.properties`](./config.example.properties).
+In order to make use of the configuration file, please copy the example to the
+same directory, fill in your sandbox user data and rename the copy to `config.properties`
```
Comment (dnl-blkv) [inline src/test/README.md]: Period in the end of sentence? :P

File: src/test/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Hi developers!
+
+
+Welcome to the bunq Java SDK integration tests. Currently we are not
+targeting the 100% test coverage, but rather want to be certain that the most
+common scenarios can run without any errors.
+
+
+## Scenarios 
+
```
Comment (dnl-blkv) [inline src/test/README.md]: We do not need this newline :)

File: src/test/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Besides these scenarios, some code of ApiContext, ApiClient and the JSON module 
+are also tested :thumbs_up:.
+
+## Configuration
+
+To run the tests you must first setup the test configuration JSON. The example
+of a configuration file is located at [`src/test/config.example.properties`](./config.example.properties).
+In order to make use of the configuration file, please copy the example to the
+same directory, fill in your sandbox user data and rename the copy to `config.properties`
+
```
Comment (dnl-blkv) [inline src/test/README.md]: We do not need this newline :)

File: src/test/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+* Make a payment to another sandbox user
+* Send a chat message in a recent payment
+* Delete the current session
+* Order a card with a second line
+
+Besides these scenarios, some code of ApiContext, ApiClient and the JSON module 
+are also tested :thumbs_up:.
+
+## Configuration
+
```
Comment (dnl-blkv) [inline src/test/README.md]: We do not need this newline :)

File: src/test/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+To run the tests you must first setup the test configuration JSON. The example
+of a configuration file is located at [`src/test/config.example.properties`](./config.example.properties).
+In order to make use of the configuration file, please copy the example to the
+same directory, fill in your sandbox user data and rename the copy to `config.properties`
+
+## Execution
+
```
Comment (dnl-blkv) [inline src/test/README.md]: We do not need this newline :)

File: src/test/README.md (hunk 1 above)
Comment (OGKevin) [inline src/test/README.md]: 😁 I mean it sooo clear that that is the end of the sentence tho 🤔 😝

**Comment by dnl-blkv** [comment]
@andrederoos

**Pull request merged**